package.json

```diff
@@ -8,15 +8,14 @@
     "elements/*"
   ],
   "scripts": {
-    "prestart": "yarn analyze",
+    "prestart": "npm run analyze",
     "start": "wsrun -p demos -c start",
     "build": "wsrun -x demos -x @openlayers-elements/testing -c build",
     "lint": "eslint --ext .ts,.js elements demos --quiet",
     "test": "wtr",
     "analyze": "polymer analyze --sources elements > demos/analysis.json",
-    "docs": "yarn build; yarn analyze; wsrun -p demos -c build",
-    "release": "changeset publish",
-    "prepare": "husky install"
```

Comment:
@spwoodcock can you also restore the `"prepare"` script?

Reply:
Thanks for picking that up!

The issue with husky was requiring `yarn` in the pre-commit hook.
I added to package.json

```
    "lint-staged": "lint-staged",
```

So we can use `npm run lint-staged` in the pre-commit hook.

Reply:
Thanks for the changes 👍 

One way also that can be used instead of having a dedicated NPM script is to use `npx lint-staged` instead in the pre-commit hook.

Reply:
I don't have `npx` installed 😅 so it makes it more versatile via npm script imo

Reply:
It's shipped with npm since many versions already :)

You can check by using `npx --version`

Reply:
My coding environment is based in a docker container (built on coder/code-server), so I only include the binaries I need for development 😄 
Awkward I know - but I only have `npm` and `pnpm` by default.

Let me know if you would rather I use the `npx` based approach though!

Reply:
I would prefer with the `npx` version, as it's included with npm version since quite some time already.

Can you check your npm version? Either it's the container that is using a too old version, or it is having some issues with the PATH?

If it's not possible for you to have it working with `npx`, then we can use the other approach since you are a contributor and I don't want to block you on working on the project ;)

Reply:
I copy the node binary and the npm scripts directly from a `node` image into my custom image, so it's not exactly a standard install: https://gitlab.com/spwoodcock/coding-env/-/blob/main/Dockerfile?ref_type=heads#L91-104

Easy enough for me to add the extra line though:

```dockerfile
ln -s /usr/local/lib/node_modules/npm/bin/npx-cli.js /usr/local/bin/npx
```

I'll do this now & update the pre-commit code to use npx 👍